### ISSUE TITLE
Is there a way to define specific compiler flags?

### ISSUE THREAD

**Issue by erip** [rgbkrk/atom-script#1095]
I'm trying to include the path to the headers of my testing library in C++ and I can't find a way to specify compiler flags. 

For example, if my headers live in `/User/erip/test/include`, I might compile my C++ program with `-I/User/erip/test/include`.

I've tried creating a profile, but my few attempts have fallen flat.

I've tried omitting the Command field (assuming it would use the default C++ command for my platform), added `-I/User/erip/test/include` to the Command Arguments field, all to no avail.

Basically whenever I had `-I...`  in my Command Arguments, I see the following:

```
bash: -/: invalid option
Usage:  bash [GNU long option] [option] ...
    bash [GNU long option] [option] script-file .
   #...
```

Is it possible to do what I'm trying to accomplish? If so, am I close? 😄 

-erip
